app/authConfig.js

```diff
@@ -10,15 +10,39 @@ const msalConfig = {
     cache: {
         cacheLocation: "sessionStorage", // This configures where your cache will be stored
         storeAuthStateInCookie: false, // Set this to "true" if you are having issues on IE11 or Edge
+    },
+    system: {
+        loggerOptions: {
+            loggerCallback: (level, message, containsPii) => {
+                if (containsPii) {	
+                    return;	
+                }	
+                switch (level) {	
+                    case msal.LogLevel.Error:	
+                        console.error(message);	
+                        return;	
+                    case msal.LogLevel.Info:	
+                        console.info(message);	
+                        return;	
+                    case msal.LogLevel.Verbose:	
+                        console.debug(message);	
+                        return;	
+                    case msal.LogLevel.Warning:	
+                        console.warn(message);	
+                        return;	
+                }
+            }
+        }
     }
 };
 
 // Add here scopes for id token to be used at MS Identity Platform endpoints.
```

Comment:
I've been meaning to change this line actually, since its misleading. Can we say something like:

"Add here the scopes that you would like the user to consent during sign-in"

Reply:
done.